test/type/config.ts

```diff
@@ -1,11 +1,14 @@
-import { useSWRConfig, Cache } from 'swr'
+import { useSWRConfig, Cache, CacheValue } from 'swr'
 import { expectType } from './utils'
 
 interface CustomCache<Data = any> extends Cache<Data> {
   reset(): void
 }
 
 export function useTestCache() {
-  expectType<Cache>(useSWRConfig().cache)
+  expectType<Map<string, CacheValue>>(useSWRConfig().cache)
+  expectType<Map<string, CacheValue>>(
+    useSWRConfig<Map<string, CacheValue>>().cache
+  )
```

Comment:
I believe we no longer want to use Map: https://github.com/vercel/swr/pull/1961/files#r872298911